src/data/grammar/flavor-text.yaml

```diff
@@ -1,46 +1,133 @@
-        "\"As #adj# as #creatureTypes.a#'s [_thingy:#itemNoun#,#abstractNoun#,#bodypart#]#_thingy#.\"",
-        "\"As the #adj# #creatureTypeOrCharacterClass# #verbStrictlyIntransitivePastTense#, #adj# #pluralNounOrMassNoun# #verbStrictlyIntransitivePastTense# around it.\"",
-        "\"#storyLocation.capitalize#'s #adj# [_role:#characterClass#]#_role.s# are the #creatureTypes.s#, yet more #adj# #_role.s# were created by [_creator:#storyLocation# itself,#character# #them#self.]#_creator#\"",
-        "\"#creatureTypeOrCharacterClass.capitalize.s#! They #verbTransitivePastTense# my favorite #itemNoun#! We'll #verbTransitive# them all!\"",
-        "\"No wonder where you find them, #creatureTypes.s# love #pluralNounOrMassNoun#.\"",
-        "\"We hold sacred the powers of #abstractNoun# and #abstractNoun#. #abstractNounMassNounOrEmotion.capitalize# and #abstractNounMassNounOrEmotion# are our greatest weapons.\" —#characterFactionOrRace.capitalize# #motto#",
-        "\"#verbTransitive.capitalize# that which was never #verbTransitivePastTense#.\" —#itemNoun.capitalize# #inscription#",
-        "\"No #abstractNoun#. No #abstractNoun#. No matter what.\" —#itemNoun.capitalize# #inscription#",
-        "\"The #placeNoun# will surround you with its #abstractNoun# if you are #adj# and #adj#.\"",
-        "\"#adj.capitalize#.\" —#anyCharacter.capitalize#",
-        "\"After #timePeriod.a# in the #placeNoun#, #character# [_adverb:seriously,nervously,foolishly,wisely,soberly,dispassionately,feverishly][_verbed:pondered,considered,examined,reviewed,weighed]#_adverb# #_verbed# the advantages of #abstractMassOrPluralNounGameConceptOrEmotion# over #abstractMassOrPluralNounGameConceptOrEmotion#.\"",
-        "\"#minorStoryLocation#, [_placeType:site,home,heart]#_placeType# of the #abstractNounOrMassNoun.capitalizeAll# #placeNoun.capitalizeAll#, source of #demonym# #abstractMassOrPluralNounGameConceptOrEmotion#.\""
-    ],
-    "occasion": ["my birthday", "my anniversary", "my first day on the job", "my last day on the job", "my bar mitzvah", "my unbirthday", "my cake day",
-                 "my inauguration", "Durin's Day", "Robanukah", "Festivus", "Wednesday", "Guy Fawkes' Day", "Boxing Day", "Shrove Tuesday", "National Corndog Day",
-                 "Ice Cream for Breakfast Day", "World Kindness Day", "Take Your #creatureTypes.capitalize# to Work Day", "Leg Day", "Casual Friday", "April Fools' Day", "CAPS LOCK DAY",
-                 "International Talk Like a Pirate Day", "the Kentucky Derby", "Christmas in July"],
-    "covering": [ "veneer", "facade", "aura", "layer", "mask", "veil", "blanket", "curtain", "girdle", "thin film", "cloud", "light dusting", "thick coating", "membrane", "halo"],
-    "motto": ["motto", "creed", "axiom", "platitude", "maxim", "epithet", "truism", "dictum", "aphorism", "adage", "prayer", "teaching"],
-    "inscription": ["inscription", "engraving"]
-  }
\ No newline at end of file
+---
```

Comment:
This is just a conversion to friendlier yaml. No new content or meaningful changes.